### DISCUSSION TITLE
[Script request] Part-DB

### DISCUSSION THREAD

**Discussion by Mightyretro** [community-scripts/ProxmoxVE#464] [Request script]
### Application Name

Part-DB

### Website

https://github.com/Part-DB/Part-DB-server

### Description

Hi,

Would love to if you could add this software to your script library

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by bvdberg01** [comment]
I can look into this one.

**Comment by MickLesk** [comment]
Merged: https://github.com/community-scripts/ProxmoxVE/pull/591
